src/components/Reports/MPGAIncomeExpensesReport/Charts/MonthlySummaryChart.tsx

```diff
@@ -0,0 +1,188 @@
+  incomeData,
+  expenseData,
+  months,
+  aspect,
+  width,
+}) => {
+  const [hasMounted, setHasMounted] = useState(false);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+
+  const combinedData = useMemo(() => {
+    const transformedExpenseData = expenseData.map((item) => ({
+      ...item,
+      monthly: item.monthly.map((value) => -Math.abs(value)),
+    }));
+
+    return [...incomeData, ...transformedExpenseData];
+  }, [incomeData, expenseData]);
+
+  const monthlyTotals = useMemo(() => {
+    const totals: {
+      [key: string]: { income: number; expense: number; total: number };
+    } = {};
+
+    combinedData.forEach((item) => {
+      item.monthly.forEach((value, index) => {
+        const month = months[index];
+        if (!totals[month]) {
+          totals[month] = { income: 0, expense: 0, total: 0 };
+        }
+
+        if (item.monthly[0] >= 0) {
+          totals[month].income += value;
+        } else {
+          totals[month].expense += value;
+        }
+      });
+    });
+
+    return months.map((month) => {
+      const { income = 0, expense = 0 } = totals[month] || {};
+      return {
+        name: month,
+        income,
+        expense,
+        net: income + expense,
+      };
+    });
+  }, [combinedData, months]);
+
+  if (!hasMounted) {
+    return null;
+  }
+
+  return (
+    <>
+      <GlobalStyles
+        styles={{
+          '.labels-print-only .recharts-label-list': { display: 'none' },
+          '@media print': {
+            '.labels-print-only .recharts-label-list': { display: 'block' },
+          },
+        }}
+      />
+      <Box className="labels-print-only">
+        <ResponsiveContainer width={`${width}%`} aspect={aspect}>
+          <BarChart
+            data={monthlyTotals}
+            margin={{ top: 15, right: 30, left: 20, bottom: 5 }}
+          >
+            <XAxis
+              dataKey="name"
+              tickFormatter={(value) => value.split(' ')[0]}
+            />
+            <YAxis
+              tickFormatter={(value) =>
+                value.toLocaleString('en-US', {
+                  style: 'currency',
+                  currency: 'USD',
+                  minimumFractionDigits: 0,
+                })
+              }
+            />
+            <Tooltip
+              formatter={(value: number | string): [string, string] => [
+                `${Number(value).toLocaleString('en-US', {
+                  minimumFractionDigits: 0,
+                  style: 'currency',
+                  currency: 'USD',
+                })}`,
+                'Total',
+              ]}
+              labelFormatter={(label: string) => `Month: ${label}`}
+            />
+            <Bar dataKey="net">
+              {monthlyTotals.map((entry, index) => (
+                <Cell
+                  key={`cell-${index}`}
+                  fill={entry.net >= 0 ? chartColors[0] : chartColors[1]}
+                />
+              ))}
+              <LabelList
```

Comment:
[Must-fix] Another hardcoded currency format in LabelList formatter. Evidence: `value.toLocaleString('en-US', { style: 'currency', currency: 'USD' })`. Impact: Consistent internationalization issue. Fix: Use `currencyFormat` utility.